portfolio/src/main/webapp/index.html

```diff
@@ -8,10 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Rania Rusly's Portfolio</h1>
+      <p>Hi I love milk and chocolate but do not like chocolate milk</p>
+      <p>Mostly she likes to eat food and sleep.</p>
+      <p>I copy pasted the second line of introduction from Stanley the cat's web and apparently it perfectly describes me too</p>
+      <img src="images/profilepic.png"/>
```

Comment:
Please start the path of the image with `/` forward slash so it will work on a development server and when you deploy to a live URL. [Ref](https://github.com/googleinterns/step/blob/master/walkthroughs/week-2-web-development/portfolio-walkthrough.md#uploading-images)